### ISSUE TITLE
Compilation problem with scanner.l

### ISSUE THREAD

**Issue by jmllorens** [HomerReid/scuff-em#30]
I am experiencing a problem while compiling the scuff_em software. After clonning the repository and running autogen.sh + configure, I get the following error:

    make[3]: se ingresa al directorio «/home/jmlloren/SOFTWARE/SCUFF_EM/scuff- em/src/libs/libMatProp/cmatheval»
    test -f scanner.c || /bin/bash ../../../../ylwrap scanner.l .c scanner.c -- /bin/bash  /home/jmlloren/SOFTWARE/SCUFF_EM/scuff-em/missing --run flex

The src/libs/libMatProp/cmatheval contains the scanner.l file, but the scanner.c file is not generated. I have tried with the bison and bison++ packages from Ubuntu 12.04. If I run the command in the folder nothing happens. I have checked that the ylwrap links to /usr/share/automake-1.11/ylwrap, in case there exists some version problems. I have tried many things, but I can not figure it out. However, I manage to compile the software form the tar.gz distribution file.

Any suggestion on how to proceed is welcome.

Jose.

**Comment by HomerReid** [comment]
In addition to 'bison,' you also need the 'flex' package to convert .l files into .c files. If you are on an ubuntu system, try:

`apt-get install flex`

and then re-attempt the compilation process.

**Comment by jmllorens** [comment]
Yes, you are right, but I forgot to mentioned that I have installed flex as well. Finally I have solved the problem by running autogen.sh and configure AFTER installing bison and flex. The first time, I installed bison and flex when make complained that yacc was not in my system. Apparently the configure script does not stop if yacc or flex are not present.

Thank you for the excellent software and for the prompt reply.
